plugins/orchestrator-common/package.json

```diff
@@ -35,7 +35,8 @@
     "clean": "backstage-cli package clean",
     "prepack": "backstage-cli package prepack",
     "postpack": "backstage-cli package postpack",
-    "openapi": "./scripts/openapi.sh"
+    "openapi": "./scripts/openapi.sh",
```

Comment:
I cloned the PR to try to generate the client and see how it works
My instinct was to go to orchestrator-backend and generate the client from there, I needed to look at the PR diff to realise it's in orchestrator-common.
Can it be moved to orchestrator-backend?

Reply:
I was told to put it in common. please raise the issue In the slack channel

Reply:
Yes it should be in common
Seems like this work is already done in this [PR](https://github.com/janus-idp/backstage-plugins/pull/1649)